go/v1/api/validators/basic_check.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Could we add a high-level description of what this lib does? For example, explaining what "basic check" means in this context, what kind of validation is considered basic.

Reply:
basic check is like a building block that can be reused by the specific packages like package, build, deployment.

added some comment.